src/closure.jl

```diff
@@ -348,7 +348,7 @@ function enclose(exorig::Expr, minbatchsize, per, threadlocal, reduction, stride
   # typexpr_incomplete is missing `funcs`
   # outerloop = Symbol("##outer##")
   num_thread_expr::Union{Symbol,Expr} = if per === :core
-    Expr(:call, min, Symbol("##NUM#THREADS##"), Expr(:call, num_cores))
+    Expr(:call, min, Symbol("##NUM#THREADS##"), Expr(:call, Static.StaticInt{Threads.nthreads()}()))
```

Comment:
this will be your unstable, right?

Reply:
It would be unstable but it's in an expression build, so it would just slap into a generated function. The real issue is that it's not pure 😅 but I'm just trying to force it to use the loopvec stuff as is